package.json

```diff
@@ -52,9 +57,13 @@
     "jest": "^26.4.2",
     "jest-watch-typeahead": "^0.6.1",
     "nodemon": "^2.0.4",
+    "npm-run-all": "^4.1.5",
```

Comment:
??

Reply:
I'll uninstall it

Reply:
Can you explain why you installed it though